### PR TITLE
add teams lifecycle

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8027

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@SerdarSoysal Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

File: Teams/teams-adoption-schedule-service-health-reviews.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ In collaboration with your technical implementation team, plan to hold monthly s
...
 - **Service health** – Call quality, meeting health and overall service health. Review of any support incidents and helpdesk ticket levels. Security and governance topics. Typically led by information protection and support teams. 
 - **Capability/project roadmap** – What is the schedule of remaining employee onboarding or further collaboration improvement projects? What additional features is your organization going to enable? Typically led by service manager or product owner for collaboration services in collaboration with key business stakeholders.
-- **Awareness campaign and training success measure update** – Includes reach, participation, key feedback themes, and open issues. Include key wins and success stories. 
+- **Awareness campaign and training success measure update** – Includes reach, participation, key feedback themes, and open issues. Include key wins and success stories.
+- **Teams lifecycle management** – includes number of active versus inactive teams, number of teams with guests users, ownerless groups, teams with duplicate names.
```
Comment (JohanFreelancer9) [inline Teams/teams-adoption-schedule-service-health-reviews.md:33]:
```suggestion
- **Awareness campaign and training success measure update** – Includes reach, participation, key feedback themes, and open issues. Includes key wins and success stories.
- **Teams lifecycle management** – includes number of active versus inactive teams, number of teams with guests users, ownerless groups, and teams with duplicate names. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/teams-adoption-schedule-service-health-reviews.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ In collaboration with your technical implementation team, plan to hold monthly s
...
 - **Service health** – Call quality, meeting health and overall service health. Review of any support incidents and helpdesk ticket levels. Security and governance topics. Typically led by information protection and support teams. 
 - **Capability/project roadmap** – What is the schedule of remaining employee onboarding or further collaboration improvement projects? What additional features is your organization going to enable? Typically led by service manager or product owner for collaboration services in collaboration with key business stakeholders.
-- **Awareness campaign and training success measure update** – Includes reach, participation, key feedback themes, and open issues. Include key wins and success stories. 
+- **Awareness campaign and training success measure update** – Includes reach, participation, key feedback themes, and open issues. Includes key wins and success stories.
+- **Teams lifecycle management** – includes number of active versus inactive teams, number of teams with guests users, ownerless groups, and teams with duplicate names.
```
Comment (JohanFreelancer9) [inline Teams/teams-adoption-schedule-service-health-reviews.md:33]:
```suggestion
- **Teams lifecycle management** – Includes number of active versus inactive teams, number of teams with guests users, ownerless groups, and teams with duplicate names. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @get-itips, one more, please.

**Comment by get-itips** [comment]
@MikePlumleyMSFT Mike, can you help me getting this merged? Thank you

**Pull request merged**